scripts/README.md

```diff
@@ -79,7 +68,7 @@ To encode relationships between incentives, see the [`relationships-README`](htt
 
 ## Utility Data
 
-`generate-utility-data.ts` reads a [dataset](https://downloads.energystar.gov/bi/portfolio-manager/Public_Utility_Map_en_US.xlsx) published by ENERGY STAR to create a mapping from ZIP codes to utilities. It writes to a CSV file in `scripts/data`, which is then imported into the SQLite database by `build.sh`. It also modifies `authorities.json` to include the utility IDs and names. This data is used in the `/api/v1/utilities` endpoint.
+`generate-utility-data.ts` reads a [dataset](https://downloads.energystar.gov/bi/portfolio-manager/Public_Utility_Map_en_US.xlsx) published by ENERGY STAR to create a mapping from ZIP codes to utilities. It writes to a CSV file in `scripts/data`, which is then imported into the SQLite database by `build.sh`. For any state with a subdirectory in `data`, it also modifies that state's `authorities.json` to include the utility IDs and names. This data is used in the `/api/v1/utilities` endpoint.
```

Comment:
This isn't specific to the authorities changes you are making here, but more for my own edification and since you are updating the relevant README portions. When I was running this utility-data script locally I kept running into this error. 
<img width="1073" alt="image" src="https://github.com/rewiringamerica/api.rewiringamerica.org/assets/1214752/ae96e94f-f7ee-4ca9-a26f-47e55f642cb6">

Or this error if I ran the script from outside the build dir.
<img width="992" alt="image" src="https://github.com/rewiringamerica/api.rewiringamerica.org/assets/1214752/ec2f0c78-edc6-46d5-9512-fed417cae8e7">


Rather than modifying the code I simply copied over the authorities.json and zip-to-utility.csv files to the build dir to make the script happy, but I was wondering if that is expected and if it should be part of the README?

Reply:
I think this is because you're compiling explicitly with `tsc`, rather than using `ts-node`, which is the expected workflow. The compiled files go into the `build` directory, which messes up script-relative paths.

Reply:
I wasn't aware that `ts-node` is our expected way to run scripts. I've always compiled with `tsc` and I know some other folks run with `tsx`.

I'm happy to standardize, but if so, I think we should document this in the top-level readme and/or the `scripts` readme. Or if we don't want to standardize, we should figure out a way a more robust way to specify the paths.